frontend/src/pages/Home/HomePage.vue

```diff
@@ -43,6 +44,10 @@ export default defineComponent({
     const fetchGems = () => store.dispatch("getGemsPendingCollectionForUser");
     const fetchFriendRequests = () => store.dispatch("getFriendRequests");
 
+    // [for testing purposes]
+    store.commit("setSelfUser", mockSelfUser);
+    store.commit("setFriends", mockFriends);
+
```

Comment:
Should this be removed?

Reply:
Ahh my injections keep getting leaking, thanks for the catch! ( i hav like 10 changed files that follow me on every branch hahaha )